blockscout-ens/bens-server/Cargo.toml

```diff
@@ -38,6 +38,6 @@ features = [
 
 [dev-dependencies]
 bens-logic = { path = "../bens-logic", features = ["test-utils"] }
-blockscout-service-launcher = { version = "0.9.0", features = [ "database-0_12", "test-server" ] }
+blockscout-service-launcher = { git = "https://github.com/blockscout/blockscout-rs", rev = "22ba428", features = [ "database-0_12", "test-server" ] }
```

Comment:
Is there any benefit in duplicating the same package in dependencies and dev-dependencies with different feature flags?

Reply:
yes, I add `test-server` feature only for tests, so I wont build `test-server` feature during prod build